app/admin/camps.rb

```diff
@@ -0,0 +1,16 @@
+ActiveAdmin.register Camp do
+  member_action :show do
+      @camp = Camp.includes(versions: :item).find(params[:id])
+      @versions = @camp.versions 
+      @camp = @camp.versions[params[:version].to_i].reify if params[:version]
+      show!
+  end
+
+  member_action :history do
+    @camp = Camp.find(params[:id])
+    @versions = PaperTrail::Version.where(item_type: 'Camp', item_id: @camp.id)
```

Comment:
in the original doc it said
    # @versions = @post.versions # <-- Sadly, versions aren't available in this scope, so use:
https://github.com/activeadmin/activeadmin/wiki/Auditing-via-paper_trail-(change-history)#adding-a-history-page